### ISSUE TITLE
Chromosome plot only supports hg19

### ISSUE THREAD

**Issue by oplantalech** [cBioPortal/cbioportal-frontend#410]
cBioPortal supports loading non-human studies by specifying the species and genome information into `portal.properties` ([see the documentation](https://github.com/cBioPortal/cbioportal/blob/v1.6.2/docs/Using-the-dataset-validator.md#validation-of-non-human-data)). However, the "chromosome plot" of the patient view seems to only support hg19: [see code here](https://github.com/cBioPortal/cbioportal-frontend/blob/4e57e2fb2747dbc9bbd59f98a5b1b5922102f994/src/pages/patientView/genomicOverview/tracksHelper.ts#L61).

**Comment by fedde-s** [comment]
As far as I understand this code, it would cause mutation or gene fusion data as well as segment-wise copy number alteration data to be visualised incorrectly on the patient view page, if a server is configured to use a different reference genome (e.g. hg38 or mm10), even if data files based on that reference genome are implied to be supported by features in the data validation/loading pipeline and utilities to update side data such as genes.

**Comment by fedde-s** [comment]
I do think that the issue you phrased is partially a back-end issue. There should be a list of genomes that *are* supported in the first place, and the server administrator should be informed somehow if the genome they specified in their `portal.properties` is invalid. The front-end issue is that, since mouse genes are explicitly supported by the rest of the system, at least one mouse genome should be supported by the patient view.

**Comment by oplantalech** [comment]
@fedde-s If you introduce a genome which is not specified [here](https://github.com/cBioPortal/cbioportal/blob/94273890828889b775ef0b7d698ab617c72edf0a/core/src/main/java/org/mskcc/cbio/portal/model/CopyNumberSegmentFile.java#L39), the validator complains. Also, if you introduce a species different from human or mouse, a message appears in the cBioPortal front page complaining that this species is not supported.

Basically, I raised the issue because the chromosome plot should support mm10.

**Comment by jjgao** [comment]
closing this. Please create a new one in the main repo if still relevant.
